types/w3c-screen-orientation/ts3.2/tsconfig.json

```diff
@@ -1,24 +0,0 @@
-{
-    "compilerOptions": {
-        "module": "commonjs",
-        "lib": [
-            "es6",
-            "dom"
-        ],
-        "noImplicitAny": true,
-        "noImplicitThis": true,
-        "strictNullChecks": false,
```

Comment:
The only difference between **TS 3.2** and **TS 3.3+** for `@types/w3c‑screen‑orientation` is whether `strictNullChecks` is enabled.